ui/raidboss/data/07-dt/alliance/jeuno-first-walk.ts

```diff
@@ -0,0 +1,792 @@
+      netRegex: { id: '9F4E', source: 'Shadow Lord', capture: false },
+      run: (data) => data.burningKeepActive = true,
+    },
+    {
+      id: 'Jeuno First Walk Shadow Lord Burning Moat',
+      type: 'StartsUsing',
+      netRegex: { id: '9F4D', source: 'Shadow Lord', capture: false },
+      delaySeconds: 0.2,
+      durationSeconds: 6,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.burningBattlementsActive)
+          return output.moatWithBattlements!();
+        if (data.burningKeepActive)
+          return output.moatWithKeep!();
+        return output.getInCircles!();
+      },
+      run: (data) => {
+        data.burningBattlementsActive = false;
+        data.burningKeepActive = false;
+      },
+      outputStrings: {
+        getInCircles: {
+          en: 'Get in circles',
+        },
+        moatWithBattlements: {
+          en: 'In circles + Close to boss',
+        },
+        moatWithKeep: {
+          en: 'In circles + Away from boss',
+        },
+      },
+    },
+    {
+      id: 'Jeuno First Walk Shadow Lord Burning Court',
+      type: 'StartsUsing',
+      netRegex: { id: '9F4C', source: 'Shadow Lord', capture: false },
+      delaySeconds: 0.2,
+      durationSeconds: 6,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.burningBattlementsActive)
+          return output.courtWithBattlements!();
+        if (data.burningKeepActive)
+          return output.courtWithKeep!();
+        return output.outOfCircles!();
+      },
+      outputStrings: {
+        outOfCircles: {
+          en: 'Out of circles',
+        },
+        courtWithBattlements: {
+          en: 'Out of circles + close to boss',
+        },
+        courtWithKeep: {
+          en: 'Out of circles + away from boss',
+        },
+      },
+    },
+    {
+      id: 'Jeuno First Walk Shadow Lord Implosion',
+      type: 'StartsUsing',
+      netRegex: { id: ['9F44', '9F45'], source: 'Shadow Lord', capture: true },
+      durationSeconds: 7,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '9F44')
+          return output.rightAndOut!();
+        return output.leftAndOut!();
+      },
+      outputStrings: {
+        leftAndOut: {
+          en: 'Go left + get out',
+        },
+        rightAndOut: {
+          en: 'Go right + get out',
+        },
+      },
+    },
+    {
+      id: 'Jeuno First Walk Lordly Shadow Implosion',
+      type: 'StartsUsing',
+      netRegex: { id: ['9F44', '9F45'], source: 'Lordly Shadow', capture: true },
+      delaySeconds: 3,
+      durationSeconds: 7,
+      alertText: (_data, matches, output) => {
+        if (matches.id === '9F44')
+          return output.rightAndOut!();
+        return output.leftAndOut!();
+      },
+      outputStrings: {
+        leftAndOut: {
+          en: 'Left of add + get out',
+        },
+        rightAndOut: {
+          en: 'Right of add + get out',
```

Comment:
```suggestion
          en: 'Left of shadow + get out',
        },
        rightAndOut: {
          en: 'Right of shadow + get out',
```
We should keep the references to the clones consistent with the other calls.

Reply:
Good point. I think I wrote this trigger separated enough from the Giga Slash triggers that I forgot about the commonality here.